src/views/tutorial-library/components/current-filters/index.tsx

```diff
@@ -17,10 +17,13 @@ interface CurrentFilterItemProps {
 	refine: (refinement: Refinement) => void
 }
 
+const IS_DEV = process.env.NODE_ENV !== 'production'
+
 function CurrentFilterItem({ refinement, refine }: CurrentFilterItemProps) {
 	const { label, type, attribute } = refinement
 
-	let labelText = label
+	let labelText
```

Comment:
Whats the context on not wanting to default to `label`?

Reply:
Oh I see, so that we can check if its undefined

Reply:
Are all of the attributes or label possibilities handled in these conditionals below? I just want to make sure we're not removing a default that is actually needed.

Reply:
> Are all of the attributes or label possibilities handled in these conditionals below? I just want to make sure we're not removing a default that is actually needed.

Yep, I'm pretty sure! I'm hoping the last test case gives us the answer here. Algolia won't do anything with a parameter it doesn't accept, and these ones listed here are all of the ones we accept AFAIK. @BRKalow can you confirm? 😊

Reply:
Yep, that's correct!